dropwizard-configuration/src/test/java/io/dropwizard/configuration/LevenshteinComparatorTest.java

```diff
@@ -11,7 +11,7 @@ class LevenshteinComparatorTest {
     private final ConfigurationParsingException.Builder.LevenshteinComparator c = new ConfigurationParsingException.Builder.LevenshteinComparator("base");
 
     /**
-     * An "java.lang.IllegalArgumentException: Comparison method violates its general contract!"
+     * A "java.lang.IllegalArgumentException: Comparison method violates its general contract!"
```

Comment:
:thought_balloon: Reading this, I "silence" the `java.lang.` part, which leads to me wanting to say "An `IllegalArgumentException` ...".

Reply:
I've amended with

```
An {@link IllegalArgumentException} "Comparison ..."
```

which is probably even better?